src/board/Board.test.jsx

```diff
@@ -0,0 +1,81 @@
+import { screen, render, fireEvent } from "@testing-library/react";
+import Board from "./Board";
+
+test("has the correct number of rows", () => {
+  render(<Board changeTurn={jest.fn()} currentPlayer={"x"} />);
+
+  expect(screen.getByText("1")).toBeVisible();
+  expect(screen.getByText("2")).toBeVisible();
+  expect(screen.getByText("3")).toBeVisible();
+  expect(screen.getByText("4")).toBeVisible();
+  expect(screen.getByText("5")).toBeVisible();
+  expect(screen.getByText("6")).toBeVisible();
+  expect(screen.getByText("7")).toBeVisible();
+  expect(screen.getByText("8")).toBeVisible();
+  expect(screen.getByText("9")).toBeVisible();
+});
+
+describe("when user clicks a cell", () => {
+  test("board is updated with correct marker", () => {
+    const mockChangeTurn = jest.fn();
+    const mockShowAlert = jest.fn();
+    const mockCloseAlert = jest.fn();
+    render(
+      <Board
+        showAlert={mockShowAlert}
+        closeAlert={mockCloseAlert}
+        changeTurn={mockChangeTurn}
+        currentPlayer="z"
+      />
+    );
+
+    const cellOne = screen.getByText("1");
+
+    fireEvent.click(cellOne);
+    expect(screen.queryByText("1")).not.toBeInTheDocument();
+    expect(cellOne).toHaveTextContent("z");
+  });
+
+  test("changes turn", () => {
```

Comment:
to follow up on my previous comment,  i'd only have one variable for the mocked function that you want to test. it keeps the focus on `changeTurn` and indicates that it doesn't really matter what the values of the other props are.   
```
  const mockChangeTurn = jest.fn();

    render(
      <Board
        showAlert={jest.fn()}
        closeAlert={jest.fn()}
        changeTurn={mockChangeTurn}
        currentPlayer="z"
      />
    );

    const cellOne = screen.getByText("1");
    fireEvent.click(cellOne);
    
    expect(mockChangeTurn).toHaveBeenCalled();
  });
  ```
  
Part of the reason why i'm mentioning this is because for my current client code base, there is an excessive amount of variables set to jest.fn() but they don't do anything; they end up being more of a distraction and it's hard to know which part actually matters